src/schema/users_model.py

```diff
@@ -0,0 +1,38 @@
+from datetime import datetime, timezone
+
+from pydantic import BaseModel, EmailStr, Field
+
+from src.schema.questions_model import Level
+
+
+class User(BaseModel):
+    """
+    Model to hold  user data.
+    """
+
+    name: str = Field(min_length=10)
```

Comment:
can we separate these fields? first name and last name? some people may not put a space btw. i'm sure front end can concat their input n generate the gull name

Reply:
done